aten/src/ATen/native/native_functions.yaml

```diff
@@ -3345,6 +3345,13 @@
   dispatch:
     DefaultBackend: celu_
 
+- func: siamese_loss(Tensor input1, Tensor input2, Tensor label, float margin=1.0, int reduction=Mean) -> Tensor
+  use_c10_dispatcher: fill
```

Comment:
This line is no longer needed, as the full c10 dispatcher is used by default.

Reply:
Oh alright.
Wouldn't line 3348 be necessary though?

Reply:
Yep, line 3348 looks good!